platform/android/java/lib/src/main/java/org/godotengine/godot/GodotRenderView.java

```diff
@@ -69,6 +68,6 @@ public interface GodotRenderView {
 	 */
 	default boolean canCapturePointer() {
 		// Pointer capture is not supported on native XR devices.
-		return !DeviceUtils.isNativeXRDevice(getView().getContext()) && getInputHandler().canCapturePointer();
+		return getInputHandler().canCapturePointer();
```

Comment:
@dsnopek I found that this change introduces a small regression when using the XR editor on Android XR with mouse and keyboard.
I have a fix which depends on https://github.com/GodotVR/godot_openxr_vendors/pull/392.

Reply:
Switching back to `draft` until v4.2.1-stable for the vendors plugin is out.